sdf/1.4/inertial.sdf

```diff
@@ -1,34 +1,77 @@
 <!-- Inertial -->
 <element name="inertial" required="0">
-  <description>The inertial properties of the link.</description>
+  <description>
+    The link's mass, position of its center of mass, and its central inertia
+    properties.
+  </description>
 
   <element name="mass" type="double" default="1.0" required="0">
     <description>The mass of the link.</description>
   </element>
 
   <element name="pose" type="pose" default="0 0 0 0 0 0" required="0">
-    <description>This is the pose of the inertial reference frame, relative to the link reference frame. The origin of the inertial reference frame needs to be at the center of gravity. The axes of the inertial reference frame do not need to be aligned with the principal axes of the inertia.</description>
+    <description>
+      This pose (translation, rotation) describes the position and orientation
+      of the link's center-of-mass-frame C relative to the link-frame L.
+      The first three components (x y z) specify the position vector from Lo
+      (the link-frame origin) to Co (the link's center of mass) as
+      `x L̂x + y L̂y + z L̂ᴢ`, where L̂x, L̂y, L̂ᴢ are link-frame L's orthogonal unit
+      vectors. The subsequent values characterize C's orientation relative to
+      link-frame L as a sequence of Euler rotations
+      (r p y) documented in http://sdformat.org/tutorials?tut=specify_pose,
+      or as a quaternion (x y z w), where w is the scalar component.
```

Comment:
We only support quaternions in >= SDFormat 1.9

Reply:
Please tell me the quaternion is given as (w x y z)!

Reply:
To be consistent with ROS and Ignition/Gazebo, we went with xyzw https://github.com/ignitionrobotics/sdf_tutorials/pull/56#issuecomment-919274182

Reply:
fixed in https://github.com/ignitionrobotics/sdformat/pull/990/commits/2a62ad1f2e3df1bd66a6a9b5c90688f589ab493d

Reply:
:( but consistency is a good argument. Eigen stores it that way also but was apparently so ashamed that they hid it by having their Quaternion constructor take w,x,y,z!

Reply:
I hear your pain, but at least the convention is specified in the `@rotation_format` attribute as `quat_xyzw`.